doc/source/point-in-time-recovery.rst

```diff
@@ -0,0 +1,84 @@
+.. _pitr:
+
+Point-in-Time Recovery
+********************************************************************************
+
+|PITR| is restoring a database up to a specific moment. |PITR| includes restoring the data from a backup snapshot and replaying all events that occurred to this data up to a specified moment from :term:`oplog slices <Oplog slice>`. |PITR| helps you prevent data loss during a disaster such as crashed database, accidental data deletion or drop of tables, unwanted update of multiple fields instead of a single one.
+
+|PITR| is available in |PBM| starting from v1.3.0. |PITR| is enabled via the ``pitr.enabled`` config option.
+
+.. code-block:: bash
+
+   $ pbm config --set pitr.enabled=true
+
+Incremental backups
+================================================================================
+
+When |PITR| is enabled, |pbm-agent| starts slicing the :term:`oplog <Oplog>`. The |pbm-agent| to do the slicing is elected in the same way as secondary nodes elect a new primary. (Find more information in :ref:`pbm.architecture.agent`.)
+
+A slice covers a 10 minute span of oplog events. It can be shorter if |PITR| is disabled or if slicing is stopped by a backup operation.
```

Comment:
```suggestion
A slice covers a 10 minute span of oplog events. It can be shorter if |PITR| is disabled or interrupted by the start of a backup snapshot operation.
```